test/format/wts.c

```diff
@@ -93,11 +93,7 @@ wts_open(const char *home, int set_api, WT_CONNECTION **connp)
 	    "create,checkpoint_sync=false,cache_size=%" PRIu32 "MB",
 	    g.c_cache);
 
-#ifdef _WIN32
-	p += snprintf(p, REMAIN(p, end), ",error_prefix=\"t_format.exe\"");
```

Comment:
Did you test to see whether it's safe to make this change? I suspect the original code was there for a reason.